Document/0x07a-Testing-Authentication-and-Session-Management.md

```diff
@@ -66,6 +66,98 @@ If any of these two conditions raise an issue, reject the request and do not all
 
 
 
+### Testing JSON Web Token (JWT)
+
+#### Overview
+
+JSON Web Token (JWT) ensures the integrity and secure transmission of information within a JSON object between two parties and is defined in RFC 7519<sup>[1]</sup>. For mobile apps it's more and more used to authenticate both, the message sender and receiver.
```

Comment:
> JSON Web Token (JWT) ensures the integrity and secure transmission

It doesn't really ensure the "secure transmission" in the sense that it does not provide confidentiality.

> For mobile apps it's more and more used to authenticate both, the message sender and receiver.

I think this is better without the comma: "both the message sender and receiver".